source/gmsh/utilities.cc

```diff
@@ -119,7 +119,10 @@ namespace Gmsh
   }
 #endif
 
+  // explicit instantiations
+#ifdef DEAL_II_WITH_OPENCASCADE
 #include "utilities.inst"
+#endif
 }
```

Comment:
Can you update the whole header guard please?
Otherwise users run into silly undefined references the moment they try to use these utilities for GMSH when OpenCASCADE is not found.